### DISCUSSION TITLE
How does `rabbitmq-diagnostics` authenticate to RabbitMQ nodes?

### DISCUSSION THREAD

**Discussion by afifurrohman-id** [rabbitmq/rabbitmq-server#11973] [Other] [answered]
### Is your feature request related to a problem? Please describe.

When starting rabbitmq without default username and password using `rabbitmq-diagnostics -q ping` to check is fail, because can't authenticate

### Describe the solution you'd like

There may any cli parameter to add like `-u --username` `-p --password` `--password-stdin`

### Describe alternatives you've considered

read env: `RABBITMQ_DEFAULT_USER`, `RABBITMQ_DEFAULT_PASS`

**Comment by afifurrohman-id** [comment]
My bad.

If someone in future search how to authenticate while health check: https://stackoverflow.com/questions/69893966/how-to-check-rabbitmq-connectionhealth-check-up-or-not

**Comment by michaelklishin** [comment, marked as answer]
@afifurrohman-id there is a [dedicated section in the CLI guide](https://www.rabbitmq.com/docs/cli#erlang-cookie).
